charts/sonarqube/templates/sonarqube-sts.yaml

```diff
@@ -136,7 +136,7 @@ spec:
               awk 1 /tmp/props/sonar.properties /tmp/props/secret.properties > /tmp/result/sonar.properties
             fi
           volumeMounts:
-          {{- if .Values.sonarProperties }}
+          {{- if or .Values.sonarProperties .Values.sonarSecretKey ( not .Values.elasticsearch.bootstrapChecks) }}
```

Comment:
Maybe a suggestion here since we are repeating this condition, maybe we could add an helper in _helpers.tpl file such as isSonarConfigDefined